docs/source/cpp/compute.rst

```diff
@@ -1816,7 +1816,7 @@ Structural transforms
   The output type is an Array of items for the ``FIRST``/``LAST`` options
   and an Array of List of items for the ``ALL`` option.
 
-* \(6) Extract a child value based on a sequence of indices passed in
+* \(6) Extract a child value based on a sequence of indices and the field names passed in
```

Comment:
"or a field name"?

@jorisvandenbossche What do you think about this change?

@Divyansh200102 Could you add examples for a field name case like existing examples for a sequence of indices case?

Reply:
Maybe "a field name or a sequence of indices".  I agree that we should add examples.